en/advanced-topics/hot-update-manager.md

```diff
@@ -0,0 +1,253 @@
+# Hot Update AssetsManager
+
+This document will fully cover the `AssetsManager` module for hot update, includes technical details and usage. As the requirements of the hot update process for developers may be different, and each developer may also face different problems. So that developers need to fully understand the details of the hot update mechanism to be able to customize the workflow to meet their needs.
+
+So this document is relatively long, it tries to introduce the hot update mechanism gradually, but will not introduce too much user level code. you want to first understand how to use the hot update mechanism to update their game developers, you can try our [simple tutorial](hot-update.md) to get started.
+
+## Introduction to Asset Hot Update
+
+Asset hot update is designed for the game to dynamically update assets, where the asset can be textures, audio and even game logic. During the maintenance and operations of the game, you will be able to upload new assets to your server so that your game keeps in sync of the remote server and automatically downloads new assets to the user's device. In this way, new designs, new play experience and even new game content will be immediately pushed to your users hands. It is important that you don't have to publish and submit your application to each market and go through a painful application submission review!
+
+The asset hot update manager has walked through three important phases:
```

Comment:
Delete this whole section. We are in the v3 docs, no need to talk about the history when they should just always use the latest version.

Reply:
Good point.